compiler-core/src/format/tests.rs

```diff
@@ -5549,3 +5549,67 @@ fn pipeline_inside_tuple_is_not_nested_if_only_item() {
 "#
     );
 }
+
+// github.com/gleam-lang/gleam/issues/2608
+#[test]
+fn comments_are_not_moved_out_of_lists() {
+    assert_format!(
+        r#"fn main() {
+  [
+    1, 2,
+    // list
+  ]
+
+  [
+    wibble,
+    wobble,
+    // list
+  ]
+}
+"#
+    );
```

Comment:
There's still two things being tested in this test!

Reply:
Oops fixed it!